olive/passes/openvino/conversion.py

```diff
@@ -55,38 +70,51 @@ def _run_for_config(
         config: Dict[str, Any],
         output_model_path: str,
     ) -> OpenVINOModelHandler:
-        import torch
-
         try:
-            from openvino.runtime import serialize
-            from openvino.tools.mo import convert_model
+            import openvino as ov
         except ImportError:
             raise ImportError("Please install olive-ai[openvino] to use OpenVINO model") from None
 
-        # output model always has ov_model as name stem
-        model_name = "ov_model"
+        if model.framework == Framework.SNPE:
+            raise ValueError("OpenVINO conversion is not supported for SNPE model.")
 
-        model_input_param = "input_model"
         input_model = model.model_path
-        if model.framework == Framework.TENSORFLOW and Path(model.model_path).is_dir():
-            model_input_param = "saved_model_dir"
         if model.framework == Framework.PYTORCH:
             input_model = model.load_model()
 
-        if config.get("extra_config") is None:
-            config["extra_config"] = {}
+        example_input = None
+        if config.get("example_input_func"):
+            example_input_func = config["example_input_func"]
+            if isinstance(example_input_func, str):
```

Comment:
nit: There is no need for the condition here. We can always directly use `example_input = self._user_module_loader.call_object(example_input_func)`. The `UserModuleLoader` object can call a function directly too https://github.com/microsoft/Olive/blob/main/olive/common/user_module_loader.py#L26

Reply:
Will do